Jenkinsfile

```diff
@@ -154,20 +157,20 @@ messagePrefix = "Jenkins ${env.JOB_NAME} build: <${env.BUILD_URL}display/redirec
 try {
     timestamps {
         def containers = [
-          [os: 'centos6',    arch: 'x86_64', flavor: 'server',  variant: ''],
-          [os: 'opensuse',   arch: 'x86_64', flavor: 'server',  variant: ''],
-          [os: 'opensuse',   arch: 'x86_64', flavor: 'desktop', variant: ''],
-          [os: 'opensuse15', arch: 'x86_64', flavor: 'desktop', variant: ''],
-          [os: 'opensuse15', arch: 'x86_64', flavor: 'server',  variant: ''],
-          [os: 'centos7',    arch: 'x86_64', flavor: 'desktop', variant: ''],
-          [os: 'xenial',     arch: 'amd64',  flavor: 'server',  variant: ''],
-          [os: 'xenial',     arch: 'amd64',  flavor: 'desktop', variant: ''],
-          [os: 'bionic',     arch: 'amd64',  flavor: 'server',  variant: ''],
-          [os: 'bionic',     arch: 'amd64',  flavor: 'desktop', variant: ''],
-          [os: 'debian9',    arch: 'x86_64', flavor: 'server',  variant: ''],
-          [os: 'debian9',    arch: 'x86_64', flavor: 'desktop', variant: ''],
-          [os: 'centos8',   arch: 'x86_64', flavor: 'server',  variant: ''],
-          [os: 'centos8',   arch: 'x86_64', flavor: 'desktop', variant: '']
+          [os: 'centos6',    arch: 'x86_64', flavor: 'server',  variant: '',    package_os: 'CentOS 6'],
+          [os: 'opensuse',   arch: 'x86_64', flavor: 'server',  variant: '',    package_os: 'OpenSUSE'],
+          [os: 'opensuse',   arch: 'x86_64', flavor: 'desktop', variant: '',    package_os: 'OpenSUSE 15'],
```

Comment:
Double-checking; should this be OpenSUSE or OpenSUSE 15?

Reply:
Yes, thanks. I'll update this on master.